enterprise/backup/src/main/java/org/neo4j/restore/RestoreDatabaseCommand.java

```diff
@@ -75,6 +80,18 @@ public void execute() throws IOException, CommandFailed
         checkLock( databaseDir.toPath() );
 
         fs.deleteRecursively( databaseDir );
-        fs.copyRecursively( fromPath, databaseDir );
+        if ( !databaseDir.equals( transactionLogsDirectory ) )
```

Comment:
We should make sure that the paths are canonicalized (and symlinks are resolved) before doing this comparison. Ponder if for example:

```
dbDir = /a/b/c
txDir = /a/b/c/d/..
```

comparison would fail, and we'd try to delete the same dir twice. 

The canonicalization might be done elsewhere but it's best to be explicit *here* in my opinion